src/VkLayer_GF_amber_scoop/include/vk_layer_amber_scoop/vulkan_commands.h

```diff
@@ -0,0 +1,146 @@
+// Copyright 2020 The gf-layers Project Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#ifndef VK_LAYER_AMBER_SCOOP_VULKAN_COMMANDS_H
+#define VK_LAYER_AMBER_SCOOP_VULKAN_COMMANDS_H
+
+#include <vulkan/vulkan.h>
+
+#include <cstdint>
+
+#include "vk_layer_amber_scoop/vk_deep_copy.h"
```

Comment:
Please move the header files alongside `amber_scoop_layer.cc`. We can treat these an "internal" headers. My intention for include/ is for public headers, but I don't think we will have any of these for Vulkan layers.

Reply:
And you will just do `#include "vk_deep_copy.h"`

Reply:
@paulthomson Moving headers to same directory with .cpp files causes some problems. As far as I know, "llvm-header-guard" expects header files to be located in a directory that has "include" somewhere in the path. Otherwise "llvm-header-guard" uses full path of the file to generate the header guard name, like this: `_HOME_ILKKAS_GIT_GF_LAYERS_SRC_VKLAYER_GF_AMBER_SCOOP_SRC_VK_DEEP_COPY_H`. Of course I can disable that check, but I think that's still a valuable check. Also "cpp-lint" wants the directory name when including header files: `#include "vulkan_commands.h"` causes error: `./src/VkLayer_GF_amber_scoop/src/amber_scoop_layer.cc:33:  Include the directory when naming .h files  [build/include_subdir]`. However that error can be avoided with `#include "./vulkan_commands.h"`

Reply:
Ah thanks for checking! Leave them as they are, but can you rename the vk_layer_amber_scoop/ to VkLayer_GF_amber_scoop/. I know this might violate some naming convention, but I think it will be the best solution.